ui/page/privacy/setup_privacy_page.go

```diff
@@ -151,7 +151,19 @@ func (pg *SetupPrivacyPage) privacyIntroLayout(gtx layout.Context) layout.Dimens
 // Part of the load.Page interface.
 func (pg *SetupPrivacyPage) HandleUserInteractions() {
 	if pg.toPrivacySetup.Clicked() {
-		pg.ChangeFragment(NewSetupMixerAccountsPage(pg.Load, pg.wallet))
+		accounts, err := pg.wallet.GetAccountsRaw()
+		if err != nil {
+			log.Error(err)
+		}
+		// We are using 2 here because of imported wallet.
+		if accounts.Count <= 2 {
```

Comment:
I believe it will be more health to check what the 2 wallets are. 

```suggestion
		if accounts.Count <= 2 && accounts.Acc[n] == "imported" {
```
this is just a random code suggestion.

Reply:
I didn't want to waste the extra computational resources looping through accounts slice, since the Count element already hold the len of the slice. So unless there is a way to delete the "imported" wallet, accounts.Count should always be => 2.